### PR TITLE
Created a template for writing issues 

### PR DESCRIPTION
**Pull request by Lukan707**
Following Helges requirements and proposals,
with the addation of writing a short precise description as a title, instead of a user story.

### CODE REVIEW

**Review by AlbinoLoaf — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue-template-.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+about: Template for creating issues
+title: User story or short precise description
+labels: ''
+assignees: ''
+
+---
+
+##User story
+Write it from the user perspective (end-user, admin) if it's a feature.
+Write if from the developers (or equal) perspective, if it's a change to the code architechture or
```
Comment (SkarpKat) [inline .github/ISSUE_TEMPLATE/issue-template-.md:12]: Typo it should be "it" not "if" i think.

**Review by SkarpKat — CHANGES_REQUESTED** [review]
There is one thing that needs to be changed before it's merged. I know it's small but let's do it before so we don't have to do it again.

File: .github/ISSUE_TEMPLATE/issue-template-.md (hunk 1 above)
Comment (Lukan707) [reply .github/ISSUE_TEMPLATE/issue-template-.md:12]: Yes. Nice spotted.

**Review by SkarpKat — APPROVED** [review]
Perfect

**Pull request merged**